### ISSUE TITLE
plan-all hides errors

### ISSUE THREAD

**Issue by shaharmor** [gruntwork-io/terragrunt#251]
Hey,

When using `plan-all`, if one of the internal modules has a parsing/validation error it will be hidden and not shown by `plan-all`.

Example:
For this module:
```hcl
resource "aws_route53_zone" "main" {
  name   = "${var.domain_name}"
  vpc_id = "${aws_vpc.main.id}"
}

resource "aws_route53_zone" "main" {
  name   = "${var.domain_name}"
  vpc_id = "${aws_vpc.main.id}"
}
```

There is an error because the two resources have the same name, but running `plan-all` won't show it. (Running `plan` will).

**Comment by brikis98** [comment]
What does the command do in that case? Does it show any messages? Could you paste some example log output?

**Comment by shaharmor** [comment]
The `plan` command shows this:

```
1 error(s) occurred:

* module root: 1 error(s) occurred:

* aws_route53_zone.main: resource repeated multiple times
```

`plan-all` shows this:

```
...
...
[terragrunt] [module/vpc] 2017/07/06 14:44:44 Running command: terraform plan -var-file=/env/test/vpc/../environment.tfvars
[terragrunt] [module/vpc] 2017/07/06 14:44:54 Module /env/test/vpc has finished with an error: exit status 1
[terragrunt] 2017/07/06 14:44:54 Encountered the following errors:
exit status 1
```

**Comment by brikis98** [comment]
And the error doesn't appear anywhere in the log output above?

**Comment by shaharmor** [comment]
Nope.
There is only 1 line after the plan command and I pasted it

**Comment by brikis98** [comment]
I believe this was fixed by #277.
